config/core/300-resources/metric.yaml

```diff
@@ -22,31 +22,97 @@ metadata:
 spec:
   group: autoscaling.internal.knative.dev
   versions:
-  - name: v1alpha1
-    served: true
-    storage: true
-    subresources:
-      status: {}
-    schema:
-      openAPIV3Schema:
-        type: object
-        # this is a work around so we don't need to flush out the
-        # schema for each version at this time
-        #
-        # see issue: https://github.com/knative/serving/issues/912
-        x-kubernetes-preserve-unknown-fields: true
-    additionalPrinterColumns:
-    - name: Ready
-      type: string
-      jsonPath: ".status.conditions[?(@.type=='Ready')].status"
-    - name: Reason
-      type: string
-      jsonPath: ".status.conditions[?(@.type=='Ready')].reason"
+    - name: v1alpha1
```

Comment:
If this is being generated now, can you add a comment to the top of the file indicating how that happens?

Reply:
Ah -- still worth putting a comment at the top of these files that the schema will get patched / rewritten?

Reply:
Yep, noted for monday.